src/functions/guild/editEmbed.js

```diff
@@ -0,0 +1,58 @@
+module.exports = async d => {
```

Comment:
Use prettier and follow the project's styling.

Reply:
Used prettier. Please check it.